components/Head.js

```diff
@@ -50,13 +59,13 @@ export default function HeadComponent({
       <meta itemProp="name" content={title} />
       <meta itemProp="description" content={description} />
       <meta itemProp="image" content={image} />
-      
+
       {/* Twitter Card data */}
-      <meta name="twitter:card" value="summary_large_image" />
+      <meta name="twitter:card" content="summary_large_image" />
```

Comment:
Why `value` has been changed to `content`?

Reply:
Changed this to some error , resolved this to previous one since it is not needed now .